api/base/pagination.py

```diff
@@ -231,7 +231,7 @@ def get_paginated_response(self, data):
 class PreprintContributorPagination(NodeContributorPagination):
 
     def get_resource(self, kwargs):
-        resource_id = kwargs.get('preprint_id')
+        resource_id = kwargs.get('preprint_id').split('_v')[0]
```

Comment:
* Use the delimiter in `VersionedGuidMixin`
* Maybe turn the split into a helper method that return tuple

```python
base_guid, version_number = check_versioned_guid(...)
````